helm/superset/templates/deployment-worker.yaml

```diff
@@ -33,6 +33,11 @@ spec:
     matchLabels:
       app: {{ template "superset.name" . }}-worker
       release: {{ .Release.Name }}
+  strategy:
+    type: RollingUpdate
```

Comment:
Pls make the strategy configurable. You can just add the whole `strategy` section into `values.yaml` and then call `toYaml` here.